src/sass/component/_header.scss

```diff
@@ -0,0 +1,43 @@
+.header-form {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+
+    .header-form--search-input {
+        border: none !important;
+      }
+
+    .header-form--search-button {
+        position: absolute;
+        top: 0;
+        right: 0;
+    }
+
+}
+
+
+header {
+    position: relative;
+}
+
+.mainmenu {
+    @include AU-space( border-bottom, 0.5unit solid $AU-colordark-foreground-action );
+	position: absolute;
+    width: 100%;
+    @include AU-clearfix();
+    @include AU-space( margin-top, 0.5unit);
+}
+
+.header-nav__header-link {
+    font-size: 1.25rem;
+    a {
+        text-decoration: none;
+        @include AU-space( padding, 0.5unit);
+        z-index: 200;
+    }
+    &:hover {
+        color: $AU-colordark-foreground-text;
+        text-decoration: underline;
+        text-decoration: ink;
```

Comment:
I think this is actually `text-decoration-skip: ink;`

Reply:
fixed